src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -185,4 +185,7 @@
                   ReadOnly="True"
                   Visible="False" />
 
+  <StringProperty Name="DisableCentralPackageVersions"
```

Comment:
Nit: I would have called this "EnableCentralPackageVersions" and defaulted to true - negative options are hard to parse in the human brain.

Reply:
Also nit: Once the name is settled, can this be moved to maintain alphabetical order of properties. Makes future diffing easier.

Reply:
Updated to address both of the comments.